docs/outlook/smart-alerts-onmessagesend-walkthrough.md

```diff
@@ -273,9 +273,16 @@ In this scenario, you'll add handling for sending a message. Your add-in will ch
 
 1. Add an attachment then send the message again. There should be no alert this time.
 
+## Deploy to users
+
+Similar to other event-based add-ins, add-ins that use the Smart Alerts feature must be deployed by an organization's administrator. For guidance on how to deploy your add-in via the Microsoft 365 admin center, see [Deploy to users](autolaunch.md#deploy-to-users).
```

Comment:
I was initially confused about why this section was linking to itself. Is there some way to indicate this is going to a different page? @lindalu-MSFT, any suggestions?

Reply:
Change to 

...see (Configure your Outlook add-in for event-based activation)(autolaunch.md#deploy-to-users)."

_OR_

see the Deploy to users section in (Configure your Outlook add-in for event-based activation)(autolaunch.md#deploy-to-users).

Reply:
Based on Linda Lu's comment...
```suggestion
Similar to other event-based add-ins, add-ins that use the Smart Alerts feature must be deployed by an organization's administrator. For guidance on how to deploy your add-in via the Microsoft 365 admin center, see the **Deploy to users** section in [Configure your Outlook add-in for event-based activation](autolaunch.md#deploy-to-users).
```